src/components/SEBankIDQRCode.tsx

```diff
@@ -168,13 +165,26 @@ export default function SEBankIDQrCode(props: Props) {
     language == 'sv' ? 'Öppna BankID-appen på din mobila enhet och skanna QR-koden.' :
     language == 'nb' ? 'Åpne BankID-appen på mobilenheten din og skann QR-koden.' : assertUnreachableLanguage(language);
 
+  const fullscreenHelpText = 
+    language === 'en' ? 'This is a BankID QR Code, you can click it to view it in full screen' : 
+    language == 'da' ? 'Dette er en BankID QR-kode, du kan klikke på den for at se den i fuld skærm' :
+    language == 'sv' ? 'Detta är en QR-kod från BankID. Du kan klicka på den för att visa den i fullskärm.' :
+    language == 'nb' ? 'Dette er en BankID QR-kode, du kan klikke på den for å se den i fullskjerm' : assertUnreachableLanguage(language);
```

Comment:
These should all use either a full stop or a semicolon instead of the comma.
(Swedish already does this)

Reply:
97a228c